### PR TITLE
Move the acceptance tests to follow a golden file test style

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Obviously this could be unified into a single test function now, but i decided against it in order to increase the visibility.

File: tests/run.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-py.test -s --tb=short -vv /testing/tests "$@"
+pip install pytest-golden
+
+ py.test -s --tb=short -vv /testing/tests "$@"
```
Comment (lluiscampos) [inline tests/run.sh:5]: nitpick: unnecessary space here

File: tests/run.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-py.test -s --tb=short -vv /testing/tests "$@"
+pip install pytest-golden
```
Comment (lluiscampos) [inline tests/run.sh:3]: This should be in the `Dockerfile` instead.

**Review by lluiscampos — APPROVED** [review]
Amazing! :raised_hands: 

I had to read-up a bit on `pytest-golden` but it seems to be very wise choice. And seeing the pipeline green for the first time in a while is very encouraging too :)

I haven't manually checked the golden files, obviously, so I just trust you and the tool on that.

File: tests/run.sh (hunk 2 above)
Comment (oleorhagen) [reply tests/run.sh:3]:
```
    image: mendersoftware/mender-test-containers:acceptance-testing
```
So I would have to add it to this image, which I decided against, considering the use case. But if you insist.

Comment (lluiscampos) [reply tests/run.sh:3]: Okey, I don't insist :)

**Pull request merged**